src/main/java/pages/HomePage.java

```diff
@@ -10,64 +10,47 @@
  */
 public class HomePage extends BasePage {
 
-    public HomePage(WebDriver driver) {
-        super(driver);
-    }
+    private static final String baseURL = "https://demostore.x-cart.com/";
 
-    @FindBy(css = ".input [name='login']")
-    private WebElement emailField;
+    @FindBy(className = "header_bar-sign_in")
+    private WebElement signInButton;
 
-    @FindBy(css = ".input [name='password']")
-    private WebElement passwordField;
+    @FindBy(id = "login-email")
+    private WebElement email;
 
-    @FindBy(css = ".input [name='login']")
-    private WebElement passwordConfirmationField;
+    @FindBy(id ="login-password")
+    private WebElement pass;
 
-    public String getPageUrl(){
-        return driver.getCurrentUrl();
-    }
-
-    protected void clearAndTypeToElement(String text, WebElement webElement) {
-        webElement.clear();
-        webElement.sendKeys(text);
+    public HomePage(WebDriver driver) {
+        super(driver);
     }
 
-    public HomePage openHomePage(){
-        driver.get("https://demostore.x-cart.com/");
-        driver.manage().window().maximize();
+    public HomePage openHomePage() {
+        driver.get(baseURL);
         return this;
     }
 
-//    public HomePage openSignUpForm(){
-//        WebElement signInButton = driver.findElement(By.cssSelector(".header_bar-sign_in [type='button']"));
-//        signInButton.click();
-//        return this;
-//    }
-
-    public HomePage enterRegistrationInfo(String email, String password, String passwordConfirmation){
-        enterEmail(email);
-        enterPassword(password);
-        enterPasswordConfirmation(passwordConfirmation);
+    public HomePage openSignInForm(){
+        waitForElementClickable(signInButton).click();
         return this;
     }
 
-    public HomePage submitRegistration(){
-        passwordConfirmationField.submit();
-        return PageFactory.initElements(driver, HomePage.class);
-    }
-
-    public void enterEmail(String emailAddress) {
-        emailField.sendKeys(emailAddress);
+    public void inputLoginData(String userLogin) {
+        WebElement emailField = waitForElementClickable(email);
+        clearAndTypeToElement(userLogin, email);
     }
 
-    public void enterPassword(String password) {
-        passwordField.sendKeys(password);
+    public void inputPasswordData(String userPassword) {
+        WebElement passwordField = waitForElementClickable(pass);
```

Comment:
If you wish, you can update it to -> wait for element to be clickable and then clear + type.